.github/workflows/release.yml

```diff
@@ -8,7 +8,9 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```

Comment:
We don't need this in master branch.
We actually don't even need it in release branch, we can select branch in the actions UI

Reply:
done